documentation/using_plugins.md

```diff
@@ -265,78 +265,7 @@ google_compute_instance.app.1:
 ```
 
 ## AWS Inventory plugin
-
-The AWS Inventory plugin supports looking up running AWS EC2 instances. It supports several fields:
-
--   `profile`: The [named profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) to use when loading from AWS `config` and `credentials` files. (optional, defaults to `default`)
--   `region`: The region to look up EC2 instances from.
--   `name`: The [EC2 instance attribute](https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Instance.html) to use as the target name. (optional)
--   `uri`: The [EC2 instance attribute](https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Instance.html) to use as the target URI. (optional)
--   `filters`: The [filter request parameters](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html) used to filter the EC2 instances by. Filters are name-values pairs, where the name is a request parameter and the values are an array of values to filter by. (optional)
--   `config`: A Bolt config map where the value for each config setting is an EC2 instance attribute.
-
-One of `uri` or `name` is required. If only `uri` is set, then the value of `uri` will be used as the `name`.
```

Comment:
I'm a little unclear if this should start with "Both" or "Either"?
Should it read...

1) Both `uri` and `name` are required. If only `uri` is set then the value of `uri` will be used for `name`.
OR
2) Either `uri` or `name` is required. If only `uri` is set then the value of `uri` will be used for `name`.

Reply:
This file is deleted and moved to https://github.com/puppetlabs/puppetlabs-aws_inventory/pull/1/commits/e55e64bd7d7265dd42ff58d7ad84b403c39cb304 . Will translate this suggestion.